nltk/tokenize/texttiling.py

```diff
@@ -101,7 +100,7 @@ def tokenize(self, text):
         # implementation states that it offers no benefit to the
         # process. It might be interesting to test the existing
         # stemmers though.
-        #words = _stem_words(words)
+        # words = _stem_words(words)
```

Comment:
I think the space non-space after the `#` in this line was intended. Please remove the space here.

One possible way to differentiate code in comment is to use `##...` instead of `#...` and keep comments as `# ...`.